corehq/apps/users/models.py

```diff
@@ -378,6 +378,7 @@ class DomainMembership(Membership):
     role_id = StringProperty()
     location_id = StringProperty()
     program_id = StringProperty()
+    show_in_ui = BooleanProperty(default=True)
```

Comment:
am sure you tested, but confirming this sets it to `True` when wrapping an object that doesn't have it coming out of the db?